### ISSUE TITLE
support for some Common Lisp reader macros

### ISSUE THREAD

**Issue by kowey** [nominolo/atto-lisp#11]
It may be useful to provide some support for a handful of [reader macros](http://www.lispworks.com/documentation/HyperSpec/Body/02_b.htm).  For example, maybe things like `#S(film :title "Jaws" :director "Steven Spielberg")` should be interpreted as though they were `(make-film :title "Jaws" :director "Steven Spielberg")`.  I don't claim to understand how they are supposed to be understood, BTW, this is just the result of typing things into SBCL and seeing what we get back

It's possible that this is thing is _so_ specific to Common-Lisp that we'll need to ask ourselves some questions about what atto-lisp is supposed to do about the existence of [multiple Lisp dialects](https://github.com/nominolo/atto-lisp/issues/10)

PS. hope this isn't spammy for you.  Should be my last thought for the day.  Thanks!
